src/applications/registry.json

```diff
@@ -517,6 +517,17 @@
       "includeBreadcrumbs": true
     }
   },
+  {
+    "appName": "Third Party Applications",
+    "entryName": "third-party-apps",
+    "rootUrl": "/third-party-applications",
+    "template": {
+      "description": "Search for third-party applications that you can connect with your VA account.",
+      "layout": "page-react.html",
+      "title": "Third-Party Applications | Veterans Affairs",
+      "vagovprod": false
```

Comment:
I'm wondering if we should leave this out of dev/staging too until we're given the green light, but I'm not sure. WE could maybe check with Stephanie about whether we're okay to publish to Dev or Staging.